Tests/Marketplace/marketplace_services.py

```diff
@@ -2411,6 +2426,23 @@ def get_packs_statistics_dataframe(bq_client):
     return packs_statistic_table
 
 
+def get_trending_packs(bq_client) -> list:
+    """
+    Updates the landing page sections data with Trending packs.
+    Trending packs: top 20 downloaded packs in the last 14 days.
```

Comment:
Definition of "Trending": top 20 packs released in the last 3 months with the most downloads in the last 14 days.

I think you are missing the `last 3 months` logic.